doc/dev/GENERATE.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Generate
+parent: Developer Documentation
+nav_order: 3
+---
+
+# How to generate the full coverage report
+
+Requires `gcovr` program and `gcc` toolchain.
+
+1. Build with `F3D_COVERAGE` option enabled.
+2. Run all tests.
+3. Generate the report with: `gcovr -r /path/to/sources --html --html-details -o coverage.html`.
+
+# How to build and test with sanitizer
+
+Requires `clang` toolchain.
+
+1. Build with `F3D_SANITIZER` option to any of the possible values.
+2. `export LSAN_OPTIONS=suppressions=/path/to/f3d/.lsan.supp:use_tls=0`.
+3. `export TSAN_OPTIONS=suppressions=/path/to/f3d/.tsan.supp`.
+4. Run all tests.
+
+# How to locally generate the jekyll based website
+
+1. Install ruby
+2. Install jekyll and all dependencies: `gem install jekyll jekyll-remote-theme jekyll-relative-links`
+3. Run jekyll locally: `~/.local/share/gem/ruby/3.0.0/bin/jekyll server`
```

Comment:
It's weird to have the full path to the executable here.  
What about `$(gem which jekyll)` or just `jekyll` and specify that the executable is located in the directory listed in `gem env`?